rails/linter-config.md

```diff
@@ -0,0 +1,182 @@
+# Using Code Linters on Rails
```

Comment:
I could see breaking these into their own sub category of Linters and Auditors.  Seems like a number of these will apply to all projects Scss, and ESLint specifically